vispy/scene/widgets/colorbar.py

```diff
@@ -138,6 +138,22 @@ def label(self, label):
         self._colorbar.label = label
 
     @property
+    def label_str(self):
+        """Get the colorbar string
+        """
+        return self._colorbar._label_str
+
+    @label_str.setter
+    def label_str(self, label_str):
+        """Set the colorbar string
+
+        Parameters
+        ----------
+        label_str : string label to be displayed with the colorbar
+        """
+        self._colorbar._label_str = label_str
+        self._update()
+
     def ticks(self):
```

Comment:
You lost the `@property` on this ticks property which is causing the errors on travis.

Reply:
Oh my... ! It's back in.

Reply:
Thanks for walking me through the process.